### PR TITLE
feat: linked data field optimization

### PR DESCRIPTION
**Pull request by zanehy**
The name field is uniformly added to the associated data #204

### CODE REVIEW

File: apioak/admin/dao/router.lua [hunk 1]
```diff
@@ -366,4 +366,141 @@ function _M.router_list_by_upstream(detail)
     return router_list
 end
 
+function _M.update_associate_upstream_name (upstream)
```
Comment (shuaijinchao) [inline apioak/admin/dao/router.lua]:
```suggestion
function _M.update_associate_upstream_name(upstream)
```

File: apioak/admin/dao/router.lua [hunk 2]
```diff
@@ -366,4 +366,141 @@ function _M.router_list_by_upstream(detail)
...
+                return "update_associate_upstream_name: update upstream name FAIL [".. update_err .."]"
+            end
+
+        until true
+    end
+
+    return nil
+end
+
+function _M.update_associate_service_name (service)
```
Comment (shuaijinchao) [inline apioak/admin/dao/router.lua]:
```suggestion
function _M.update_associate_service_name(service)
```

File: apioak/admin/dao/router.lua [hunk 3]
```diff
@@ -366,4 +366,141 @@ function _M.router_list_by_upstream(detail)
...
+                return "update_associate_service_name: update service name FAIL [".. update_err .."]"
+            end
+
+        until true
+    end
+
+    return nil
+end
+
+function _M.update_associate_plugin_name (plugin)
```
Comment (shuaijinchao) [inline apioak/admin/dao/router.lua]:
```suggestion
function _M.update_associate_plugin_name(plugin)
```

File: apioak/admin/dao/service.lua [hunk 4]
```diff
@@ -277,4 +284,57 @@ function _M.service_list_by_plugin(detail)
     return service_list
 end
 
+function _M.update_associate_plugin_name (plugin)
```
Comment (shuaijinchao) [inline apioak/admin/dao/service.lua]:
```suggestion
function _M.update_associate_plugin_name(plugin)
```

File: apioak/admin/dao/router.lua (hunk 1 above)
Comment (zanehy) [reply apioak/admin/dao/router.lua]: OK, thanks for the correction.

**Review by blust — APPROVED** [review]

**Review by shuaijinchao — APPROVED** [review]

**Pull request merged**